contrib/inventory/digital_ocean.py

```diff
@@ -147,7 +147,7 @@
 try:
     from dopy.manager import DoError, DoManager
 except ImportError as e:
-    print("failed=True msg='`dopy` library required for this script'")
+    sys.stderr.write("failed=True msg='`dopy` library required for this script'\n")
```

Comment:
same change than in previous file, we only need sys.exit(" message" ), this applies to most changes below also, sorry I should have been more explicit with last comment i made.